src/assets/sass/protocol/components/_notification-bar.scss

```diff
@@ -0,0 +1,101 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../includes/lib";
+
+.mzp-c-notification-bar {
+    @include text-display-xxs;
+    background-color: $color-gray-20;
+    border-radius: 4px;
+    border: 1px;
+    box-shadow: 1px 1px 1px rgba(27,31,35,.1);
+    margin: $layout-xs auto 0;
+    height: 42px;
```

Comment:
This fixed height might be troublesome for smaller screens and locales that have long string.